### PR TITLE
More debug printing during test.hs, when required

### PR DESCRIPTION
**Pull request by msooseth**
## Description
During test.hs, more debug printing, when it's turned on.

## Checklist

- [x] tested locally
- [ ] added automated tests
- [ ] updated the docs
- [ ] updated the changelog

### CODE REVIEW

File: test/test.hs [hunk 1]
```diff
@@ -3624,12 +3624,16 @@ checkEquiv a b = do
   if a == b then pure True else do
     when (opts.debug) $ liftIO $ putStrLn $ "Checking equivalence of " <> show a <> " and " <> show b
     x <- checkEquivBase (./=) a b True
+    when (opts.debug) $ liftIO $ putStrLn $ "UNSAT check, expect True: " <> show x
     y <- checkEquivBase (.==) a b False
+    when (opts.debug) $ liftIO $ putStrLn $ "SAT check, expect False: " <> show x
```
Comment (blishko) [inline test/test.hs:3629]:
```suggestion
    when (opts.debug) $ liftIO $ putStrLn $ "SAT check, expect False: " <> show y
```

**Review by blishko — CHANGES_REQUESTED** [review]

File: test/test.hs (hunk 1 above)
Comment (msooseth) [reply test/test.hs]: Wow, thanks, that was stupid of me

**Review by blishko — APPROVED** [review]
LGTM!

**Comment by msooseth** [comment]
Thanks for the review. Really helped actually!

**Pull request merged**